src/dotnet-uninstall/LocalizableStrings.Designer.cs

```diff
@@ -87,15 +87,6 @@ internal class LocalizableStrings {
             }
         }
         
-        /// <summary>
-        ///   Looks up a localized string similar to Failed to run as administrator.
-        /// </summary>
-        internal static string ElevationFailedExceptionMessage {
```

Comment:
nit: delete this file. This is a bug in Arcade working with VS.

Reply:
The solution does not build after this file is removed.

Reply:
ok, so maybe we do need it